docs/Configuration-Settings.md

```diff
@@ -712,6 +712,26 @@ configuration options.
   <td>Memory resource required to run a Tachyon worker. This memory does not include the memory
   configured for tiered storage.</td>
 </tr>
+<tr>
+  <td>tachyon.integration.mesos.role</td>
+  <td>*</td>
```

Comment:
set the default value for this and the other properties in the /common/src/main/resources/tachyon-default.properties file